libtenzir/builtins/operators/every_cron.cpp

```diff
@@ -218,6 +233,49 @@ class every_plugin final : public virtual operator_plugin<every_operator> {
   }
 
   auto parse_operator(parser_interface& p) const -> operator_ptr override {
+    using operator_type = scheduled_execution_operator<Scheduler>;
+    auto scheduler = Scheduler::parse(p);
+
+    auto result = p.parse_operator();
+    if (not result.inner) {
+      diagnostic::error("failed to parse operator")
+        .primary(result.source)
+        .throw_();
+    }
+    if (auto* pipe = dynamic_cast<pipeline*>(result.inner.get())) {
+      auto ops = std::move(*pipe).unwrap();
+      for (auto& op : ops) {
+        op = std::make_unique<operator_type>(std::move(op),
+                                             std::move(scheduler));
+        // Only the first operator can be a source and needs to be replaced.
+        break;
+      }
+      return std::make_unique<pipeline>(std::move(ops));
+    }
+    return std::make_unique<operator_type>(std::move(result.inner),
+                                           std::move(scheduler));
+  }
+};
+
+/// scheduler for the 'every' operator
+class every_scheduler {
```

Comment:
As a general rule of thumb, I would say "do not add a comment its both trivial and non-structuring". It's very clear from the name that this is the scheduler for `every`. Having no docs for something is sometimes perfectly okay.